src/components/paperManager.js

```diff
@@ -94,7 +94,11 @@ export default class PaperManager {
     this.paper.drawBackground({ color: defaultNodeColor });
   }
 
-  hasScheduledUpdates() {
-    return this.#paper._updates.priorities.some(updates => !util.isEmpty(updates));
+  executeAfterScheduledUpdates(callback) {
```

Comment:
This gets rid of some duplication and better encapsulates the logic of how this callback is handled.